서버 배포 오류 해결

bcrypt 모듈 설치가 원활하게 진행되지 않아 배포상 문제가 생김

``` shell
Error: Cannot find module '/home/ubuntu/cashbook-15/node_modules/bcrypt/lib/binding/napi-v3/bcrypt_lib.node'
Require stack:
- /home/ubuntu/cashbook-15/node_modules/bcrypt/bcrypt.js
- /home/ubuntu/cashbook-15/src/controllers/auth.ts
- /home/ubuntu/cashbook-15/src/router/auth.ts
- /home/ubuntu/cashbook-15/src/router/index.ts
- /home/ubuntu/cashbook-15/src/index.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/home/ubuntu/cashbook-15/node_modules/bcrypt/bcrypt.js:6:16)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Module.require (internal/modules/cjs/loader.js:974:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

- [x] 어떻게든 해결해 배포에 성공한다

--- a/package.json
+++ b/package.json
@@ -15,11 +15,11 @@
   "dependencies": {
     "@babel/core": "^7.14.6",
     "@babel/preset-env": "^7.14.7",
-    "@types/bcrypt": "^5.0.0",
+    "@types/bcryptjs": "^2.4.2",
     "@types/cors": "^2.8.12",
     "@types/validator": "^13.6.3",
     "babel-loader": "^8.2.2",
-    "bcrypt": "^5.0.1",
+    "bcryptjs": "^2.4.3",
     "class-validator": "^0.13.1",
     "cookie-parser": "^1.4.5",
     "copy-webpack-plugin": "^9.0.1",

--- a/src/controllers/auth.ts
+++ b/src/controllers/auth.ts
@@ -1,4 +1,4 @@
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
 import { Request, Response } from "express";
 import { Model } from "sequelize-typescript";
 import User from "../models/user";

